src/features/smartSearch/components/filters/SubQuery/index.tsx

```diff
@@ -127,143 +127,151 @@ const SubQuery = ({
           <Msg id={localMessageIds.examples.two} />
         </>
       )}
-      renderSentence={() => (
-        <Msg
-          id={localMessageIds.inputString}
-          values={{
-            addRemoveSelect: (
-              <StyledSelect
-                onChange={(e) => setOp(e.target.value as OPERATION)}
-                value={filter.op}
-              >
-                {Object.values(OPERATION).map((o) => (
-                  <MenuItem key={o} value={o}>
-                    <Msg id={messageIds.operators[o]} />
+      renderSentence={() =>
+        queries.length == 0 ? (
+          <Msg id={messageIds.filters.subQuery.noSmartSearches} />
+        ) : (
```

Comment:
What happens when there are no standalone smart search queries (`queries`) but there _are_ call assignments?

Reply:
This filter already takes callassignments into consideration, the `queries` variable here is an array made up of standalone queries, call assignment target queries, and call assignment "done" queries:
![bild](https://github.com/zetkin/app.zetkin.org/assets/58265097/8c55b52c-8d33-4056-b3de-7ffbe49ccf82)

Reply:
Ah, I went looking for it but clearly not thoroughly enough. Thanks for clarifying!